Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.10
```

Comment:
Please could you also add the following labels:
`copyright`: `Crown Copyright 2020`
`license`: `Apache-2.0`

Reply:
Certainly!

Reply:
Updated in PR.